ocss.sh

```diff
@@ -87,9 +90,9 @@ take_screenshot() {
   echo "Please select area"
   is_mac || sleep 0.1 # https://bbs.archlinux.org/viewtopic.php?pid=1246173#p1246173
 
-  if ! (scrot -s "$1" || screencapture -s "$1"); then #takes a screenshot with selection
+  if ! ((scrot -s "$1") || screencapture -s "$1"); then #takes a screenshot with selection
```

Comment:
Your `is_mac` changes make me think you're running this in `bash`.  Is this actually necessary if you run it in `sh`?